examples/notebooks/README.md

```diff
@@ -11,3 +11,10 @@ Be sure to set the `JUPYTER_PATH` variable in a similar way that `PYTHONPATH` is
 *Example: export JUPYTER_PATH=/path/to/carmm/folder:$JUPYTER_PATH*
 
 Alternatively, clicking on the binder link above will take you to a web platform where you can play with the notebooks without installing anything!
+
+In case you are still struggling to access and run this notebook, please do the following at the start of the notebook. 
```

Comment:
I'm not sure this reads very clearly in the flow of the README - it seems to be an additional hack appended on the end, but the flow is poor. I'll accept the pull request as it fixes the Mulliken notebook but then suggest we immediately review and just refine this README.